### ISSUE TITLE
Warning: Zend OPcache API is restricted by "restrict_api" configuration directive

### ISSUE THREAD

**Issue by Caleydon** [PrestaShop/PrestaShop#12336]
**Describe the bug**
I get an warning and blank page in the Back office when Debug mode is enabled. I tested on 2 different webhosting services with clean PrestaShop 1.7.5.0 and also with third party template.

The webhosting service told me that the reason is they are blocked following functions for security reasons:
opcache_compile_file () 
opcache_get_configuration () 
opcache_get_status () 
opcache_invalidate () 
opcache_reset () 

**To Reproduce**
1. Go to Performance tab and change settings. See the videos.

**Screenshots**
Warnings
https://ibb.co/P5VmmGC
https://ibb.co/Kz85RTR
https://ibb.co/kHSFptG

How to get an warning/blank page
https://vimeo.com/313751599/736419fc57
https://vimeo.com/313812043/81d2f9879f

**Additionnal information**
PrestaShop version: 1.7.5.0
PHP version: 7.2 (tested also with 7.1 and 5.6.)

**Comment by khouloudbelguith** [comment]
Hi @Caleydon,

Thanks for your report.
I did not manage to reproduce the issue with PS1.7.5.0.
I attached a video record.
https://drive.google.com/file/d/10MGgQaSyW7g3JlaY9j3dJ2Mu82yVPYr-/view
It's your hosting. They've disabled the opcache commands, the opcache for the configuration file had to be cleared because of the REST API. The API would make changes to the settings file, and elsewhere in the plugin PHP would include the file again, but because it's cached by the opcache it would load the old version.
Thanks!
